packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/subtoken/takeFirstPastSecondWordHarp.yml

```diff
@@ -1,12 +1,22 @@
 languageId: typescript
 command:
-  version: 1
+  version: 3
```

Comment:
In the older v1 there was no disambiguation between "first past second word" and "first two words". I updated to a newer version so that tests intent can be kept.